packages/components/fields/password-field/src/password-field.tsx

```diff
@@ -176,6 +176,8 @@ export type TPasswordField = {
    * Might be used to display additional information about the content of the field (E.g verified email)
    */
   badge?: ReactNode;
+  // Determines whether to render the "Show/Hide" button for the password field.
```

Comment:
nit: we typically use JSDoc notation for comments here

Reply:
Oh great catch! Thanks @kark - [f6691c8](https://github.com/commercetools/ui-kit/pull/2759/commits/f6691c8c8f93f96bfc0bdf1405f99dd90a419317).